### ISSUE TITLE
Taking swagger-ui from rails-assets

### ISSUE THREAD

**Issue by ktimothy** [ruby-grape/grape-swagger-rails#83]
Hey, this is a very nice project, but keeping swagger-ui dist files in the repository among with rake-task for downloading them makes me sad. I have an idea to add swagger-ui dist as a dependency in Gemfile. I can even make a PR with that. What do you think?

**Comment by dblock** [comment]
Yes! Making swagger-ui a dist gem and making it a dependency is something we'd take here.

**Comment by mleszcz** [comment]
Yeah. Update task doesn't seem to work anymore
```bundle exec rake swagger_ui:dist:update
Cloning into /tmp/swagger-ui20190806-10968-xymd0x ...
Removing files from /home/user/apps/some_app ...
Copying Javascripts ...
rake aborted!
Errno::ENOENT: No such file or directory @ apply2files - /home/user/apps/some_app/app/assets/javascripts/grape_swagger_rails
```

Would be great to be able to update swagger_ui....

**Comment by dblock** [comment]
> Yeah. Update task doesn't seem to work anymore
> 
> ```
> $ bundle exec rake swagger_ui:dist:update
> Cloning into /tmp/swagger-ui20190806-10968-xymd0x ...
> Removing files from /home/user/apps/some_app ...
> Copying Javascripts ...
> rake aborted!
> Errno::ENOENT: No such file or directory @ apply2files - /home/user/apps/some_app/app/assets/javascripts/grape_swagger_rails
> ```
> 
> Would be great to be able to update swagger_ui....

Feel free to debug and PR! Thank you.

**Comment by moskvin** [comment]
This is stale relative to the current asset strategy.

The project now vendors Swagger UI v5 bundle assets directly, tracks the embedded Swagger UI version explicitly, and provides an update task for refreshing those assets. So the original `rails-assets` motivation no longer matches the current codebase.

I'm inclined to close this as not planned in its current form. If someone wants to propose a modern switch to an external `swagger-ui-dist` style dependency, that should come back as a fresh design proposal against the current architecture.
